### PR TITLE
Fixed typos in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by schnittstabil**
Fixed typos in `CONTRIBUTING.md`
- Added composer.phar to `.gitignore` to better support the suggested workflow

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,5 +13,6 @@ zf-mkdoc-theme/
 
 clover.xml
 composer.lock
+composer.phar
```
Comment (xtreamwayz) [inline .gitignore]: Shouldn't `composer.phar` be in `.gitignore_global`?

**Comment by schnittstabil** [comment]
@xtreamwayz I don't like to have `composer.phar` in package directory either, it just is the suggested approach – I'm willing to change it to a global composer approach if the PO wants to.

However, this PR is not only about the `composer.phar` related stuff, the bugged `cd zend-stratigility` and the non-working `git clone` are fixed too.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ pull your work into the master repository. We recommend using
 3. Clone the canonical repository locally and enter it.
 
    ```console
-   $ git clone git://github.com:zendframework/zend-stratigility.git
+   $ git clone git@github.com:zendframework/zend-stratigility.git
```
Comment (weierophinney) [inline CONTRIBUTING.md]: Original was correct; most developers will not have push access to the origin repository, so read-only access is fine here; it's also faster.

File: .gitignore (hunk 1 above)
Comment (weierophinney) [inline .gitignore]: This is an unrelated change. It is up to the developer to ensure they have access to composer and do not pollute the project tree; we will, of course, reject commits that include the file when merging anyways. The files listed in `.gitignore` are those that could appear due to running packaging/QA/documentation tools, and are specific to the needs already in the repository.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ To run tests:
 - Run the tests using the "test" command shipped in the `composer.json`:
 
   ```console
-  $ composer test
+  $ ./composer.phar test
```
Comment (weierophinney) [inline CONTRIBUTING.md]: Instead of changing these all to `composer.phar`, we should be changing `composer.phar` usage to `composer`.

File: CONTRIBUTING.md (hunk 2 above)
Comment (schnittstabil) [inline CONTRIBUTING.md]: The original simply does not work; did you mean `git clone https://github.com/…`?

File: .gitignore (hunk 1 above)
Comment (schnittstabil) [inline .gitignore]: :+1: With `s/composer.phar/composer/` this is not needed anymore.

File: CONTRIBUTING.md (hunk 2 above)
Comment (weierophinney) [inline CONTRIBUTING.md]: No, I meant the `git://` protocol. Missed that there's a problem with it — the `:` following `github.com` should be a slash (`/`), and then it will work.

Comment (weierophinney) [inline CONTRIBUTING.md]: As in:

```console
$ git clone git://github.com/zendframework/zend-stratigility.git
```

**Comment by weierophinney** [comment]
Superseded by #97.

File: CONTRIBUTING.md (hunk 2 above)
Comment (schnittstabil) [inline CONTRIBUTING.md]: I see, but as far as I know, the `git://` protocol is not officially supported, [see for example, Github Help: Which remote URL should I use?](https://help.github.com/articles/which-remote-url-should-i-use/)
